index.bs

```diff
@@ -429,6 +445,23 @@ Abstract Operations {#abstract-opertaions}
     1.  Otherwise, define |object|'s [=local coordinate system=] to the [=device coordinate system=].
 </div>
 
+<h3 dfn>Accelerometer reading quantization algorithm</h3>
+
+The [=Accelerometer=] [=sensor type=] defines the following [=reading quantization algorithm=]:
+
+<div algorithm="accelerometer reading quantization">
+  : input
+  :: |reading|, a [=sensor reading=]
+  : output
+  :: A [=sensor reading=]
+
+  1. Let |quantizedReading| be |reading|.
+  1. If |quantizedReading|["x"] is not null, set |quantizedReading|["x"] to its nearest tenth.
```

Comment:
I would say "to the nearest 0.1 m/s<sup>2</sup>".

Reply:
See https://github.com/w3c/accelerometer/pull/82/commits/52a127de4940a4411b92690af50e9966de4d6c03